packages/components/pie-chip/src/chip.scss

```diff
@@ -4,6 +4,11 @@
     display: inline-block;
 }
 
+:host([disabled]),
+:host([isLoading]) {
+    pointer-events: none;
+}
+
```

Comment:
do you think we still need that if we are disabling the native elements (input and button) such as disabled=isLoading | disabled?

if so, we probably need to reflect them so they can be accessed via css and attached correctly to the react wrapper


<img width="1512" height="982" alt="Screenshot 2025-09-16 at 11 29 57" src="https://github.com/user-attachments/assets/6d406ca2-13de-4000-aaf5-77bcfed0aabb" />

https://lit.dev/docs/components/properties/#reflected-attributes